app/js/reducers/drugSearchReducer.js

```diff
@@ -0,0 +1,36 @@
+import * as actionTypes from '../actions/actionTypes';
+import mockData from '../../../__mocks__/mockData';
```

Comment:
Why do you need mockData in this reducer?

Reply:
@Betty-Kebenei We are using mock data as the initial state

Reply:
@geofrocker thanks for assisting with the explanation, 
@flacode yes, mockData was used system wide for providing mock data for usage in both tests(actions/components) and initial values for reducers